rpc_jobs/releases.yml

```diff
@@ -32,17 +32,17 @@
           repo_url: "https://github.com/rcbops/releases"
           run_issue_linker: true
           comment_body: |
-            Thank you for pull request to `rcbops/releases`!
+            Thank you for your pull request to `rcbops/releases`!
 
-            Once all `check` jobs have passed and the pull request has received a
-            sufficient number of reviews, the repository's `release` jobs can be triggered by
-            adding a comment of `:shipit:` to this pull request.
+            Once all required `check` jobs, prefixed with `check/`, have passed and the pull
+            request has received a sufficient number of reviews, the repository's `release` job
+            can be triggered by adding a comment of `:shipit:` to this pull request.
 
-            If the requested release has one or more failing tests and there are valid reasons
-            to ignore the failure, a release can be forced through by triggering the process
-            a with comment of `:shipit: skip validation`.
+            If the requested release has one or more failing project tests and there are valid
+            reasons to ignore the failure(s), a release can be forced through by triggering the
+            process a with comment of `:shipit: skip validation`.
```

Comment:
Perhaps it might benefit from the addition of:

'this option should only be used in exceptional circumstances. The first course of action should always be to fix the failing job(s)'

or similar. I just want to make it absolutely clear that it's a last resort rather than a 'oh it's release day and we haven't got time to fix anything, I'll just force it and fix it later (never)'

Reply:
`:shipit: yes I know that it's failing tests but I still really really really want to release it anyway!`